Mathlib/GroupTheory/Perm/Cycle/Factors.lean

```diff
@@ -326,10 +326,8 @@ def cycleFactorsAux [DecidableEq α] [Fintype α] (l : List α) (f : Perm α)
             (h fun h : f y = y => by
               rw [mul_apply, h, Ne, inv_eq_iff_eq, cycleOf_apply] at hy
               split_ifs at hy <;> tauto))
-      ⟨cycleOf f x::m, by
-        rw [List.prod_cons, hm₁]
-        simp,
-        fun g hg ↦ ((List.mem_cons).1 hg).elim (fun hg => hg.symm ▸ isCycle_cycleOf _ hx) (hm₂ g),
+      ⟨cycleOf f x::m, by simp [List.prod_cons, hm.1],
```

Comment:
Since you're changing this line already, would you mind writing `cycleOf f x :: m` instead for readability?

Reply:
```suggestion
      ⟨cycleOf f x :: m, by simp [List.prod_cons, hm.1],
```